src/app/components/ModalShoppingList.jsx

```diff
@@ -146,6 +147,64 @@ export default class ModalShoppingList extends TranslatedComponent {
     }
   }
 
+  /**
+ * Send all blueprints to EDOMH. This is a modified copy of registerBPs because this.state.blueprints was empty when I tried to modify sendToEDEng and I couldn't figure out why
+ * @param {Event} event React event
+ */
+  sendToEDOMH(event) {
+    event.preventDefault();
+    const ship = this.props.ship;
+    let blueprints = [];
+
+    //create the json
+    for (const module of ship.costList) {
+      if (module.type === 'SHIP') {
+        continue;
+      }
+      if (module.m && module.m.blueprint) {
+        if (!module.m.blueprint.grade || !module.m.blueprint.grades) {
+          continue;
+        }
+        if (module.m.blueprint.special) {
+          blueprints.push({
+            "item": module.m.symbol,
+            "blueprint": module.m.blueprint.special.edname
+          });
+        }
+        for (const g in module.m.blueprint.grades) {
+          if (!module.m.blueprint.grades.hasOwnProperty(g)) {
+            continue;
+          }
+          if (g < module.m.blueprint.grade) {
+            continue;
+          }
+          blueprints.push({
+            "item": module.m.symbol,
+            "blueprint": module.m.blueprint.fdname,
+            "grade": module.m.blueprint.grade,
+            "highestGradePercentage":1.0
+          });
+        }
+      }
+    }
+
+    //create JSON to encode
+    let baseJson = {
+      "version":1,
+      "name":ship.name, // TO-DO: Import build name and put that here correctly
+      "items": blueprints
+    }      
+    
+    let JSONString = JSON.stringify(baseJson)
+    let deflated = zlib.deflateSync(JSONString)
+    
+    //actually encode
+    let link = base64url.encode(deflated)
+    link = "edomh://coriolis/?" + link;
+
+    window.open(link, "_self")
+  }
+
```

Comment:
Code looks good! This could looks like it might also work for edsy. This is just SLEF, right?

Reply:
It isn't SLEF, EDOMH uses its own format :\

Reply:
SLEF wouldn't fit inside the url :)